SDALib-CPP/SDAReplay/ReplayMain.cpp

```diff
@@ -1,11 +1,19 @@
 #include "ReplayDriver.h"
 
-int main()
+int main(int p_argc, char **p_path)
 {
-    std::string replayFileName("ReplayData/bbRecording20220330-123618.txt");
+    std::string replayFileName;  // initialize the string
+    for (int i = 1; i < p_argc; i++)
+    {
+        // picks the argument written after --bbfile
+        if (p_path[i] == std::string("--bbfile"))
+        {
+            replayFileName = p_path[++i];
```

Comment:
what happens if there are spaces in the path?

Reply:
works fine, one space, multiple space, it doesn't break it. Probably the main function has an build in that any extra spaces get ignored. BTW I couldn't seem to react on your last remark about the main file, that was an accident, I used it for testing, it has no use in the project.